scripts-test/weather.py

```
@@ -0,0 +1,60 @@
+
+Dependencies:
+    - requests: For making HTTP requests to the weather API
+
+Note:
+    The script uses the free wttr.in API service which may have rate limits.
+    Internet connection is required for the script to work.
+"""
+
+import requests
+
+def get_weather(city):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_weather`

(ANN201)